spec/classes/plugin/puppetdb_spec.rb

```diff
@@ -19,6 +19,16 @@
 
       it { should compile.with_all_deps }
       it { should contain_foreman__plugin('puppetdb').with_package(package_name) }
+      it do
+        should contain_foreman_config_entry('puppetdb_enabled')
+          .with_value(true)
+          .that_requires(['Class[foreman::database]', 'Foreman::Plugin[puppetdb]'])
+      end
+      it do
+        should contain_foreman_config_entry('puppetdb_address')
+          .with_value('https://localhost:8081/pdb/cmd/v1')
+          .that_requires(['Class[foreman::database]', 'Foreman::Plugin[puppetdb]'])
+      end
```

Comment:
```suggestion
      it do
        should contain_foreman_config_entry('puppetdb_enabled')
          .with_value(true)
          .that_requires(['Class[foreman::database]', 'Foreman::Plugin[puppetdb]'])
        should contain_foreman_config_entry('puppetdb_address')
          .with_value('https://localhost:8081/pdb/cmd/v1')
          .that_requires(['Class[foreman::database]', 'Foreman::Plugin[puppetdb]'])
      end
```
I think this is preferred

Reply:
Given this is a cherry pick, I don't want to change the code. But if you would like please do open a PR to the main branch with this update.